.idea/Springboard.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't check in `.idea`
https://stackoverflow.com/questions/11968531/what-to-gitignore-from-the-idea-folder